src/tic-tac-toe.behavior.js

```diff
@@ -62,18 +60,20 @@ export default System.be({
 
   // Handle player moves
   makeMove: {
-    select: { self: DB.self, currentPlayer, gameStatus, event },
+    select: { self: DB.self, currentPlayer, event },
     where: [
       { Case: [DB.self, 'currentPlayer', currentPlayer] },
-      { Case: [DB.self, 'gameStatus', gameStatus] },
+      { Case: [DB.self, 'gameStatus', 'ongoing'] },
       { Case: [DB.self, `/on/click`, event] },
-      { Is: [gameStatus, 'ongoing'] },
```

Comment:
This change was not strictly necessary, however I suspect that maybe it was on obvious that this simplified version would have worked. In this case query only matches when `gameStatus` is `ongoing` which I assume was the intention here.